bindings/python/tests-py/test_jsonschema.py

```diff
@@ -65,6 +65,12 @@ def test_from_str_error():
         JSONSchema.from_str(42)
 
 
+def test_tuple():
+    schema = {"properties": {"foo": {"type": "array"}}}
+    instance = {"foo": (1, 2, 3)}
+    assert is_valid(instance, schema) == True
```

Comment:
As the default behavior for type mismatch is to consider the input valid, it would be nice to have a test case that will examine the situation when the input is not valid.

Reply:
with out this change, this test throws:

```
ValueError: Unsupported type: 'tuple'
```

Reply:
Ahh I now see what you mean, `is_valid({}, schema) is True` weird, Ill fix this.